testsuite/sclang/TestInteger.sc

```diff
@@ -0,0 +1,29 @@
+TestInteger : UnitTest {
+	test_forby_with_positive_integer_arguments {
+		var forby_arguments = List.new;
```

Comment:
and in sclang code we definitely prefer camelCase

Reply:
Done.